lib/schedule/gtfs/stop_time.ex

```diff
@@ -8,6 +8,8 @@ defmodule Schedule.Gtfs.StopTime do
   alias Schedule.Gtfs.Stop
   alias Schedule.Gtfs.Trip
 
+  @type by_id :: %{Trip.id() => [t()]}
```

Comment:
What do you think about calling this `by_trip_id`? When I read it I think it's going to be by stop time ID.